src/modules/add.js

```diff
@@ -0,0 +1,43 @@
+// modules/add.js
+export const newScore = () => {
+  const nameInput = document.querySelector('#input-name');
+  const scoreInput = document.querySelector('#input-score');
+  const name = nameInput.value;
+  const score = scoreInput.value;
+
+  if (name === '' || score === '') {
+    // eslint-disable-next-line no-alert
```

Comment:
- [ ] [Optional] I've noticed that you have disabled ESLint using inline comments. It's important to note that this is generally not considered a good practice. I strongly recommend that you remove all ESLint disable comments and instead go through the linter output, addressing each error individually. ✅🔍 Linters play a crucial role in identifying potential issues and enhancing code consistency. By addressing these linter errors, you'll improve the readability and maintainability of your code. Keep up the great work! 🚀💻